src/main/scala/com/gd/twitteranalytics/util/TweetsConstant.scala

```diff
@@ -0,0 +1,11 @@
+package com.gd.twitteranalytics.util
+
+object TweetsConstant {
+
+  val PARTITION_COLUMN = "event_date"
+  val SAVEPATH = "twitter-analytics/dataLake/eng-tweets"
```

Comment:
should be supplied via config

Reply:
Okay, will move these to properties file .